src/js/components/replenishment/ReplenishmentCheckPage.jsx

```diff
@@ -210,7 +210,7 @@ class ReplenishmentSecondPage extends Component {
       this.props.showSpinner();
       const url = `/api/replenishments/${this.props.match.params.replenishmentId}`;
       const payload = { status: 'COMPLETED' };
-      apiClient.post(url, flattenRequest(payload))
+      apiClient.put(url, flattenRequest(payload))
```

Comment:
I'm ok with either post or put here, but we can tech-huddle this if necessary. Lazy developers like myself will exclusively use GET and POST when testing an API so I don't mind supporting that.

https://stackoverflow.com/questions/33452765/what-to-do-when-rest-post-provides-an-id

However, there are some resources where this would be confusing and unexpected so we should definitely be conscious of what we're doing and explicit / restrictive where possible.

Here's another long-winded argument for supporting either approach. 

https://stackoverflow.com/questions/630453/what-is-the-difference-between-post-and-put-in-http

I don't yet know where I stand.